[BE-117] 레코드 수정

kdomo opened the pull request:
## 관련 이슈 번호
- [BE-117 / 레코드 수정](https://recodeit.atlassian.net/browse/BE-117)

## 설명
레코드 수정 테스트와 로직 작성하였습니다.


## 변경사항

<!-- PR에 반영되어야 할 변경 사항들을 가능한 자세히 작성 해주세요. -->
<!-- - [x] 회원가입 및 로그인 -->

## 질문사항

kdomo committed:
[BE-117] test: Record 수정 테스트코드 작성

kdomo committed:
[BE-117] feat: Record 수정

kdomo committed:
[BE-117] fix: 레코드 수정 swagger response Long으로 지정

Jaeyeop-Jung commented on src/main/java/com/recordit/server/controller/RecordController.java:
```diff
@@ -129,4 +131,28 @@ public ResponseEntity deleteRecord(
+			)
+	})
+	@PutMapping("/{recordId}")
+	public ResponseEntity<Long> updateRecord(
```
modifyRecord는 어떤가요?

Jaeyeop-Jung commented on src/main/java/com/recordit/server/dto/record/UpdateRecordRequestDto.java:
```diff
@@ -0,0 +1,46 @@
+	private List<String> deleteImages;
+
+	@Builder
+
```
띄워쓰기 있네용

Jaeyeop-Jung commented on src/main/java/com/recordit/server/dto/record/UpdateRecordRequestDto.java:
```diff
@@ -0,0 +1,46 @@
+
+	@Builder
+
+	public UpdateRecordRequestDto(String title, String content, String colorName, String iconName,
```
코드 컨벤션 지켜주세용

Jaeyeop-Jung commented on src/main/java/com/recordit/server/service/ImageFileService.java:
```diff
@@ -106,7 +106,12 @@ public List<String> getImageFiles(
+	public void deleteAttachmentFiles(
+			@NonNull RefType refType,
+			@NonNull Long refId,
+			@NonNull List<String> attachmentFileNames) {
```
) 여기 한칸 띄어쓰기 해주세용

Jaeyeop-Jung commented on src/main/java/com/recordit/server/service/ImageFileService.java:
```diff
@@ -106,7 +106,12 @@ public List<String> getImageFiles(
+			@NonNull RefType refType,
+			@NonNull Long refId,
+			@NonNull List<String> attachmentFileNames) {
+		imageFileRepository.deleteAllByRefTypeAndRefIdAndSaveNameIsIn(refType, refId, attachmentFileNames);
```
이거 쿼리 메소드 SaveNameIsIn할 때 IsIn으로 해야되나요?

kdomo replied:
`IsIn`이 아닌 `In`으로 사용해도 되네용

kdomo committed:
[BE-117] fix: 코드리뷰 수정
- updateRecord -> modifyRecord 로 변경
- UpdateRecordRequestDto -> ModifyRecordRequestDto 로 변경
- 코드 컨벤션 수정
- deleteAllByRefTypeAndRefIdAndSaveNameIsIn -> deleteAllByRefTypeAndRefIdAndSaveNameIn 으로 변경

kdomo committed:
[BE-117] fix: Record Entity에 update -> modify 로 변경

kdomo committed:
Merge branch 'develop' into feature/BE-117

kdomo committed:
[BE-117] chore: merge conflict 해결

kdomo committed:
[BE-117] chore: merge conflict 해결
